docs/topics/sgx.md

```diff
@@ -50,13 +50,14 @@ spec:
     image: <IMAGE>
     command: <COMMAND>
     imagePullPolicy: IfNotPresent
-    volumeDevices:
-    - devicePath: /dev/sgx
-      name: dev-sgx
+    volumeMounts:
+    - name: dev-sgx
+      mountPath: /dev/sgx
     securityContext:
       privileged: true
   volumes:
   - name: dev-sgx
     hostPath:
       path: /dev/sgx
+      type: CharDevice
```

Comment:
@mdemri just for my edification could you explain what this means.

Reply:
It means that the mapped host path is connected to a Character device (SGX device is this kind of device, afaik)
Also in IBM cloud its defined this way
https://cloud.ibm.com/docs/services/data-shield?topic=data-shield-deploying

See Kubernetes hostPath documentation
https://kubernetes.io/docs/concepts/storage/volumes/#hostpath